constants/src/lib.rs

```diff
@@ -22,6 +22,38 @@ lazy_static! {
     pub static ref DEFAULT_APP_NAME: String =
         format!("{}+{}", DRIVER_SHORT_NAME, DRIVER_METRICS_VERSION.as_str());
     pub static ref DRIVER_ODBC_VERSION: String = format_driver_version();
+    pub static ref KEYWORDS: String = [
+        "AGGREGATE".to_string(),
+        "ARRAY".to_string(),
+        "BINDATA".to_string(),
+        "BOOL".to_string(),
+        "BOOLEAN".to_string(),
+        "BSON_DATE".to_string(),
+        "BSON_TIMESTAMP".to_string(),
+        "DBPOINTER".to_string(),
+        "DEPTH".to_string(),
```

Comment:
I see [DEC](https://github.com/10gen/mongosql-rs/blob/921ea1b6df6d2d9df360c12bcd49a46acddadacc/mongosql/src/parser/mongosql.lalrpop#L780) in mongosql.

Reply:
DEC is in the ODBC reserved keywords list already!

Reply:
You are right! I totally missed it!